src/gluonts/distribution/poisson.py

```diff
@@ -0,0 +1,109 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+# Standard library imports
+from typing import Dict, Optional, Tuple, List
+
+# Third-party imports
+import numpy as np
+
+# First-party imports
+from gluonts.model.common import Tensor
+from gluonts.core.component import validated
+
+# Relative imports
+from .distribution import Distribution, _sample_multiple, getF, softplus
+from .distribution_output import DistributionOutput
+
+
+class Poisson(Distribution):
+    r"""
+    Poisson distribution, i.e. the distribution of the number of
+    successes in a specified region.
+
+    Parameters
+    ----------
+    mu
```

Comment:
This parameter is commonly denoted as `lambda`. I think it would be better to stick to that naming, what do you think?

Reply:
Hi, I think lambda is better too, I will change it

Reply:
But lambda is a reserved word, should I just put it like _lambda?

Reply:
oh right, you can call it `rate` then